src/modules/local_position_estimator/BlockLocalPositionEstimator.cpp

```diff
@@ -112,9 +112,10 @@ BlockLocalPositionEstimator::BlockLocalPositionEstimator() :
 	_ref_alt(0.0)
 {
 #if defined(ENABLE_LOCKSTEP_SCHEDULER)
-	// For lockstep 250 Hz are needed because ekf2_timestamps need to be
-	// published at 250 Hz.
+	// For lockstep 250 Hz are needed because lockstep needs to be
+	// updated at that rate.
 	_sensors_sub.set_interval_ms(4);
```

Comment:
It might be better to drop the interval entirely. I'd still like to bring the SITL rate more in line with most setups (perhaps 250 Hz -> 400 Hz).

Reply:
Removed